.github/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Connect Telegram Bot
```

Comment:
File must be .github/workflows/build.yml

Reply:
fixed